packages/react-native/React/Views/UIView+React.h

```diff
@@ -115,7 +115,7 @@ typedef struct {
  * The anchorPoint property doesn't work in the same way as on web - updating it updates the frame.
  * To work around this, we take both the transform and the transform-origin, and compute it ourselves
  */
-@property (nonatomic, assign) CATransform3D reactTransform;
+@property (nonatomic, copy) NSArray* reactTransform;
```

Comment:
Would `retain` be sufficient here if we make sure callers do not hold on to the transform?

The implementation's already not copying anyway, so this is currently not true :)

Reply:
Yup, fixed!